submodules/enterprise_hierarchy/version.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3.0, <1.6.0"
```

Comment:
please update this to `required_version = ">= 1.3.0, <1.7.0"` in preparation for migration to opentofu